briskedu/brisk-learning
# Refresh Chrome extension content when toggling between documents

### Summary:
When we change documents, we should close the current extension content and fetch new content once the page loads.

https://app.asana.com/0/1203963408726580/1203963408926855

### Test Plan:
1. Load an assignment with two students
2. Confirm when moving to the next assignment, info first collapses and then expands with new content
